lib/Common/Memory/Recycler.cpp

```diff
@@ -3173,10 +3173,19 @@ Recycler::SweepWeakReference()
     RECYCLER_PROFILE_EXEC_BEGIN(this, Js::SweepWeakPhase);
     GCETW(GC_SWEEP_WEAKREF_START, (this));
 
+#ifdef GCETW
+    uint scannedCount = 0;
+    uint keptCount = 0;
+#endif
+
     // REVIEW: Clean up the weak reference map concurrently?
     bool hasCleanup = false;
-    weakReferenceMap.Map([&hasCleanup](RecyclerWeakReferenceBase * weakRef) -> bool
+    weakReferenceMap.Map([&hasCleanup, &scannedCount, &keptCount](RecyclerWeakReferenceBase * weakRef) -> bool
```

Comment:
This will need to be under an `#ifdef` as well otherwise the variables won't exist, right?